nion/swift/ExportDialog.py

```diff
@@ -28,13 +29,16 @@
 from nion.utils import Converter
 from nion.utils import Geometry
 from nion.utils import Model
-
+from nion.utils import Observable
 
 if typing.TYPE_CHECKING:
     from nion.swift.model import DisplayItem
 
 _ = gettext.gettext
 
+PIXELS_PER_INCH = 96.0
+PIXELS_PER_CM = 37.795275591
+
```

Comment:
These don't seem to be used anymore. These values should go into the `ConversionUnits` enum.